Realm/ObjectServerTests/RLMObjectServerTests.m

```diff
@@ -20,6 +20,20 @@
 
 #import "RLMSyncUser+ObjectServerTests.h"
 
+static NSURL *makeRealmURL(const char *function, NSString *identifier) {
+    // 'function' is expected to be an Objective-C method name: "[MyClass fooBarBaz]"
```

Comment:
URLComponents seems like a good idea. I'd like to keep the `identifier` if possible, since I have some tests in the pipeline that would use it.